deps/loader.pl

```diff
@@ -0,0 +1,20 @@
+% TODO are there any other file loading goals we need to expand???
+goal_expansion(use_module(library(File)), use_module(deps/Path)) :-
+  dep(File, Path).
+goal_expansion(use_module(library(File), ImportList), use_module(deps/Path, ImportList)) :-
+  dep(File, Path).
+goal_expansion(consult(File), consult(deps/Path)) :-
+  must_be(atom, File),
```

Comment:
This isn't quite right. It might be an atom or a path which is atoms around `/` operators. This should be more like "when not a list"